index.js

```diff
@@ -53,30 +53,46 @@ module.exports = function(options, cb) {
     function(next) {
       var params = {
         clusterArn: options.clusterArn,
-        taskDefinitionArn: options.taskDefinitionArn,
-        containerName: options.containerName,
         cmd: options.cmd,
-        endOfStreamIdentifier: endOfStreamIdentifier
+        containerName: options.containerName,
+        endOfStreamIdentifier: endOfStreamIdentifier,
+        env: options.env,
+        startedBy: options.startedBy,
+        taskDefinitionArn: options.taskDefinitionArn
       }
 
       taskRunner.run(params, next);
     }
   ], function(err, taskDefinition) {
     if (err) return cb(err);
 
-    var taskArn = taskDefinition.tasks[0].taskArn;
-    var taskId = taskArn.substring(taskArn.lastIndexOf('/')+1);
+    const taskArn = taskDefinition.tasks[0].taskArn,
+          taskId  = taskArn.substring(taskArn.lastIndexOf('/')+1),
+          formatter = new FormatStream();
 
-    var formatter = new FormatStream();
-    var logs = new LogStream({
+    const logs = new LogStream({
       logGroup: logOptions['awslogs-group'],
       logStream: `${logOptions['awslogs-stream-prefix']}/${options.containerName}/${taskId}`,
       endOfStreamIdentifier: endOfStreamIdentifier
     });
-    
+
     var stream = combiner(logs, formatter);
     stream.logStream = logs;
 
+    process.on('SIGINT', () => {
```

Comment:
Listening for the interrupt signal should probably happen in the `./bin/ecs-task-runner` script. Since this `index.js` is intended to be used for module use.